src/actions/index.js

```diff
@@ -11,8 +11,10 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return { type: INCREMENT, payload: +1 }
```

Comment:
This is great!

However, why did you choose to have a payload here? You could have just incremented by 1 in the reducer and then you wouldn't need to have the second property on your action object.

Alternatively, if you'd been wanting the potential to increment/decrement by other amounts, then you could have had your action creator take a an argument and then pass that argument to the payload.

ie.

```
export const increment = (num) => {
  // Fill in this function
  return { type: INCREMENT, payload: num}
```